README.md

```diff
@@ -1,9 +1,13 @@
 # Project MONAI
 **M**edical **O**pen **N**etwork for **AI**
 
-[![License](https://img.shields.io/badge/License-Apache%202.0-green.svg)](https://opensource.org/licenses/Apache-2.0)
+[![License](https://img.shields.io/badge/license-Apache%202.0-green.svg)](https://opensource.org/licenses/Apache-2.0)
 [![CI Build](https://github.com/Project-MONAI/MONAI/workflows/build/badge.svg?branch=master)](https://github.com/Project-MONAI/MONAI/commits/master)
 [![Documentation Status](https://readthedocs.org/projects/monai/badge/?version=latest)](https://monai.readthedocs.io/en/latest/?badge=latest)
+[![codecov](https://codecov.io/gh/Project-MONAI/MONAI/branch/master/graph/badge.svg)](https://codecov.io/gh/Project-MONAI/MONAI)
+[![Total alerts](https://img.shields.io/lgtm/alerts/g/Project-MONAI/MONAI.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/Project-MONAI/MONAI/alerts/)
+[![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/Project-MONAI/MONAI.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/Project-MONAI/MONAI/context:python)
```

Comment:
this language grade badge doesn't work now, looks like their server is unstable

Reply:
Yes, the server is not stable now, the code should be correct.